### ISSUE TITLE
Using the ng2-dragula on ngx-datatable columns

### ISSUE THREAD

**Issue by HackPoint** [valor-software/ng2-dragula#933]
I've opened a question on SO:
[My Question regarding this issue](https://stackoverflow.com/questions/53930855/ngx-datatable-integrate-with-dragula-on-columns-instead-of-the-native-ngx-dnd)

I'm using angular5 and it's impossible to update it for this moment.
Can ng2-dragula be integrated into the ngx-datatable?

**Comment by razorhenry1** [comment]
Yup. This works but after drag, table item order is not maintained. This will help for start.
https://codesandbox.io/s/8lk1yp92r8
